ms2/src/org/labkey/ms2/Protein.java

```diff
@@ -446,21 +466,79 @@ void addPeptide(Range range, Integer newLevel, int curIdx, int curRowStart, int
                     if(!_forCoverageMapExport)
                     {
                         mass = getSequenceMass(_sequence.substring(range.start,(range.start + range.length)));
-                        details = String.format("Mass: %.2f  \nTotal Scans: %d ", mass, counts.countScans);
+                        details = new StringBuilder(String.format("Mass: %.2f <br/> Total Scans: %d", mass, counts.countScans));

```

Comment:
countScans is only relevant for the MS2 usage. Can we hide it for the Panorama usage? I know it's been there for a long time, but it's useless.